tests/foreman/cli/test_import.py

```diff
@@ -539,11 +544,204 @@ def test_import_users_recovery(self, test_data):
             ssh.command(u'ls ${HOME}/.transition_data/users*').stdout[:-1]
         )
         for record in transition_data:
-            self.assertNotEqual(User.info({'id': record['sat6']}).stdout, '')
+            self.assertEqual(
+                User.info({'id': record['sat6']}).return_code, 0
+            )
         Import.user({'csv-file': files['users'], 'delete': True})
         # do the cleanup
         ssh.command(u'rm -rf {}'.format(' '.join(pwdfiles)))
 
+    @data(*import_org_data())
+    def test_import_host_collections_default(self, test_data):
+        """@test: Import all System Groups from the default data set
+        (predefined source) as the Host Collections.
+
+        @feature: Import Host-Collections
+
+        @assert: 3 Host Collections created
+
+        """
+        files = dict(self.default_dataset[1])
+        files['users'] = update_csv_values(
+            files['users'],
+            test_data,
+            self.default_dataset[0]
+        )
+        files['system-groups'] = update_csv_values(
+            files['system-groups'],
+            test_data,
+            self.default_dataset[0]
+        )
+        # import the prerequisities
+        self.assertEqual(
+            Import.organization({'csv-file': files['users']}).return_code, 0
+        )
+        self.assertEqual(
+            Import.host_collection(
+                {'csv-file': files['system-groups']}
+            ).return_code, 0
+        )
+        transition_data = csv_to_dataset(
+            ssh.command(
+                u'ls ${HOME}/.transition_data/organizations*'
+            ).stdout[:-1]
+        )
+        # now to check whether the all HC from csv appeared in satellite
+        imp_orgs = csv_to_dataset([files['users']])
+        [imp_org.update(
+            {'sat6': tr['sat6']}
+        ) for tr in transition_data for imp_org in imp_orgs
+            if tr['sat5'] == imp_org['organization_id']]
```

Comment:
@Ichimonji10: I wasn't really satisfied with my solution....but it was alive :)
Thanks for suggesting use of product, I'll update the code.

for the other comment:
I'm always out of ideas in coming up with 'descriptive' names for iterator variables, especially for something as abstract as `transition_data`.
Subjectively, the most descriptive and straight-forward name for me is and always will be `i` as for `iterator`